osfoffline/settings/defaults.py

```diff
@@ -17,7 +17,7 @@
 POLL_DELAY = 5 * 60  # seconds (set to 1 min now for QA purpose, #todo change it back)
 
 # Time to keep alert messages on screen (in milliseconds); may not be configurable on all platforms
-ALERT_TIME = 1000  # ms
+ALERT_TIME = 10000  # ms
```

Comment:
Please change this back to 1 second for now.

We display a lot of alerts. With this change, each individual alert appears for 10 seconds (during which interval many alerts drop out of the queue and aren't displayed at all)